crates/viewer/re_view_time_series/src/view_class.rs

```diff
@@ -213,6 +217,15 @@ impl ViewClass for TimeSeriesView {
                 .extend(maybe_visualizable.iter().cloned());
         }
 
+        // Ensure we don't modify this list anymore before we check the `suggested_filter`.
+        let indicated_entities = indicated_entities;
```

Comment:
neat trick, should do that more often

Reply:
Ahaha, it's me being lazy and keeping the diff small. Introducing a scope would be even cleaner at the cost of an additional nesting level.